erica/infrastructure/worker/huey_task.py

```diff
@@ -0,0 +1,5 @@
+from erica.config import get_settings
+
+from huey import RedisHuey
+
+huey_task = RedisHuey('erica-huey-queue', url=get_settings().queue_url, immediate=get_settings().use_immediate_worker)
```

Comment:
Can we call this just `huey`? It's an instance of a RediHuey, so calling it a task confused me. 

Also: did you consider using dependency injection to obtain the Huey instance? Since it's imported at the module level _and_ because we can use the real huey (with the immediate setting), I'm not sure that it would have any benefit at all, I'm just curious if you thought about it.

Reply:
Also the module naming convention seems to be `erica.infrastructure.$libraryName....`. Can we rename this one to `erica.infrastructure.huey`?

Reply:
> Can we call this just huey? It's an instance of a RediHuey, so calling it a task confused me.

Yes. I changed it to huey_task because I wanted to prevent overloading huey but I

> Also: did you consider using dependency injection to obtain the Huey instance? Since it's imported at the module level and because we can use the real huey (with the immediate setting), I'm not sure that it would have any benefit at all, I'm just curious if you thought about it.

Yes, I thought about it and you are describing my thought process😄. I could not see a good way to use dependency injection without making the code more complicated and because I did not see a clear benefit for either testing or different environments, I decided that the simpler version might be better

> Also the module naming convention seems to be erica.infrastructure.$libraryName..... Can we rename this one to erica.infrastructure.huey

We can.